index.js

```diff
@@ -240,7 +240,7 @@ self.uhtml = (function (exports) {
                   } // this node has no meaning in the future list, so it's more than safe
                   // to remove it, and check the next live node out instead, meaning
                   // that only the live list index should be forwarded
-                  else parentNode.removeChild(get(a[aStart++], -1));
+                  else if(get(a[aStart], -1).parentNode = parentNode) parentNode.removeChild(get(a[aStart++], -1));
```

Comment:
you are assigning + this is not a solution, is a workaround to an underlying issue that will mess up with everything else.